### ISSUE TITLE
Product Variations not working correctly.

### ISSUE THREAD

**Issue by jamessawle2** [facebookincubator/facebook-for-woocommerce#768]
We have just noticed that product variations are not working on our website. we have compared ones that work with ones that do not and the only reason we can see id that it has something to do with this plugin.

For the ones that work we have noted that they are synced correctly with Facebook and have a 'fb_product_group_id' under the custom fields section along with an ID number. 

In all of the products that do not work the 'fb_product_group_id' is missing and there is also a notice stating that 'This product is not yet synced to Facebook.' on the side bar when editing the product.

We have updated the plugin to the latest version (1.9.7) and manually set off a re-sync but this has made no difference. we have tried disabling the plugin but again it does not seem to make a difference.

We have manually check all of the products that do not show variations when visiting the shop pages and all that do not work are all that have not synced with facebook. 

Is there a way to manually force a single product to sync?

and how do we get the fb_product_group_id to be added to the product page when it is not? bu the way other facebook custom fields are there, i.e. fb_image, fb_product_description, and fb_visibility.

This page **does not** work: [http://www.celtic-moments.com/product/chicken-rock/](http://www.celtic-moments.com/product/chicken-rock/)

This page **does** work: [http://www.celtic-moments.com/product/port-moar/](http://www.celtic-moments.com/product/port-moar/)

Just noticed that there are only 149 products listed on Facebook Page whereas there are 345 Products on the website.

Again the only difference between ones that do work and ones that dont is this field missing.

please help.

James

**Comment by jieminz22** [comment]
## Automated Support Message

Hello,

Thanks for your interest in our plugin! Our developer team tries to provide personal support on GitHub whenever possible.

If you have a feature request or feedback for the developers, reply with `Feature Request` or `Feedback`.
If you are a developer or have a question for the developers related to code, reply with `Developer Question`.
If you are running ads with us, reply with `Advertiser`.

If you have encountered a bug or a problem with the plugin, keep reading for information that can help you resolve the issue yourself or provide us details we need to help you.

## Search Existing Issues

First, please [search our issues](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) to see if your problem has been reported before.
There may already be a workaround or a solution to your problem. You can use the [search bar](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) near the top of our issues page. Be sure to search both _open_ and _closed_ issues.
If you do find a similar issue, leave a comment on that issue instead of creating a new one. Your comment helps us prioritize which existing issues are important and urgent.

## Common Resolution Methods

If you are experiencing a problem or bug and have not found any similar issue, please try the following:

* Upgrade to the latest version of the plugin
* Try to `Resync` products using the button on the plugin config page
* Reset the plugin by selecting `Reconfigure > Advanced Options > Delete`, and setting up this plugin again
* Try deactivating other plugins that you think may be interfering, then reset and set up again

## Necessary Information

If the above steps did not help, please provide the following information in your next comment in order for us to debug your problem:

* WooCommerce version
* WordPress version
* Plugin version (please note: we only support up-to-date versions of the plugin)
* A screenshot of where you see the problem (for example: if it is on Facebook, provide a screenshot of Facebook, if it is on WooCommerce, provide a screenshot of WooCommerce. You can provide multiple screenshots if the problem relates to different screens.)
* The URL of your website
* The URL of your Facebook Page
* The URL(s) of where you took your screenshot(s)
* If applicable, detailed steps you took in order to see the problem

Please include `Debug Ready` in your reply so our automated system can note your issue for manual review. Please be aware that due to a high volume of issues, we may not be able to respond immediately.

## Response Time

We prioritize issues with the tags `Advertiser` or `Multiple Reports`.
For issues that have the tags `Debug Ready`, `Feature Request` or `Developer Question`, we aim to respond within 30 days.

If you have not responded to our automated message with one of the above phrases, such as `Debug Ready`, we may close your issue after 30 days. If a `Feature Request` is inactive (without new requests, messages or pull requests) for 6 months, we may close it.

Please note that we are not always able to provide support for issues related to interaction from other plugins with this plugin or related to the Facebook advertising interface. To get support related to Facebook ads, try visiting the [Facebook Ads Help Center](https://www.facebook.com/business/help).

For more information on our response times, visit [Facebook Support on GitHub](https://github.com/facebookincubator/facebook-for-woocommerce/wiki/Facebook-Support-on-GitHub).

Thank you,
The Facebook Dev Team

**Comment by Martystl** [comment]
Did you get your issue resolved?  I have not and have similar issue.  67 items and 11 variations per item for a total of 737 variations.  It has been over 6 weeks to no avail.  My issue # is 728 and website is www.martyknightphotography.com

**Comment by jamessawle2** [comment]
Hi, sorry no fix yet, support is a tad rubbish, definitely a plug-in issue, have heard from 8 others with same issue.

James

**Comment by jamessawle2** [comment]
Still no fix, even disabling the plugin does not fix the issue, there are many data fields missing that are related to the plugin and because of these WooCommerce does not display the items correctly. Nearly two weeks nd still no fix, also now up to 24 people that have emailed me with the same problems, their fixes are not really useful, they have basically removed plugin and removed all items from store and then re-added them again to fix the issue. There must be a fix, waiting on the developer to come back to me, hopefully with something worth trying.

**Comment by jieminz22** [comment]
## Automated Support Message - Issue Closed

Hello,

Because it has been 30 days since this issue was updated and you have not included `Debug Ready` in your reply, our automated system will now close your issue.
If you are still having this problem, please open a new issue and reference this one.

For more information on our response times and support policy, visit [Facebook Support on GitHub](https://github.com/facebookincubator/facebook-for-woocommerce/wiki/Facebook-Support-on-GitHub).

Thank you,
The Facebook Dev Team

**Comment by jamessawle2** [comment]
Till not fixed, but hay you closed it to make it look better I suppose

**Comment by Martystl** [comment]
Agree, issue is NOT fixed.
